server/routes/summary_routes.ts

```diff
@@ -114,14 +123,13 @@ export function registerSummaryAssistantRoutes(
         question: req.body.question,
       });
       try {
-        let result = response.body.inference_results[0].output[0].result;
-        result = JSON.parse(result).output.text;
-        return res.ok({ body: result });
+        return res.ok({ body: response.body.inference_results[0].output[0].result });
```

Comment:
So both agents(tools): `LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID` and `SUMMARY_AGENT_CONFIG_ID` has changed the output from plain text to json?

Reply:
Yeah. Only `os_insight` agent has change its output structure from json to text. Because there is a post function added in connector:
```
"post_process_function": "return params.output.text;"
```